computer_player_script.py

```diff
@@ -5,13 +5,15 @@
 from core_tic_tac_toe.human_player import HumanPlayer
 from core_tic_tac_toe.game import Game
 from core_tic_tac_toe.game_logic import GameLogic
+from core_tic_tac_toe.input_validator import InputValidator
 
 if __name__ == "__main__":
     io = Io()
     display = Display(io)
     board = Board()
     game_logic = GameLogic()
-    player_1 = HumanPlayer("Player 1", "X", board, display)
+    input_validator = InputValidator()
+    player_1 = HumanPlayer("Player 1", "X", board, display, input_validator)
```

Comment:
not a big deal, but I noticed you have a lot of arguments for `HumanPlayer`. I think this is a smell, so I wanted to point it out. https://refactoring.guru/smells/long-parameter-list

Reply:
Agreed! I plan on refactoring this class to remove the unnecessary dependencies.